app/src/main/java/com/humberto/tasky/core/database/AgendaDatabase.kt

```diff
@@ -19,12 +22,19 @@ import com.humberto.tasky.core.database.entity.TaskEntity
         ReminderEntity::class,
         AttendeeEntity::class,
         PhotoEntity::class
-               ],
-    version = 1)
+    ],
+    version = 2,
+    autoMigrations = [
+        AutoMigration(from = 1, to = 2, spec = AgendaDatabase.Migration1To2::class)
+    ]
+)
```

Comment:
Tip - while in development, you can completely destroy your DB since all the data is test data anyways. You don’t need to be concerned about users’ data being lost 😄 Once the app is released to the public, then your approach is the way to go, but until that stage, staying at version 1 is fine.

Reply:
I actually tried deleting the app multiple times but was getting migration errors, I don't know why...